framework/src/modules/chain/chain.js

```diff
@@ -194,6 +174,25 @@ module.exports = class Chain {
 
 			self.logger.info('Modules ready and launched');
 
+			this.channel.subscribe(
+				'network:subscribe',
```

Comment:
This name still doesn't seem quite right. We should raise another issue to adjust the naming.

Reply:
Actually we can use this existing issue: https://github.com/LiskHQ/lisk-sdk/issues/3263